Adapter_oM/Settings-Config/AdapterSettings.cs

```diff
@@ -20,15 +20,17 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
-namespace BH.Adapter
+namespace BH.oM.Adapter
 {
-    public class AdapterConfig
+    public class AdapterSettings
     {
         public bool HandleDependencies { get; set; } = true;
         public bool ProcessInMemory { get; set; } = false;
         public bool UseAdapterId { get; set; } = true;
         public bool CloneBeforePush { get; set; } = true;
         public bool WrapNonBHoMObjects { get; set; } = false;
+        public bool UseHashComparerAsDefault { get; set; } = true;
```

Comment:
This will change the behaviour of all the structural adapters unless overridden in the constructors. Are we sure we want to default this to true before it has been tested to work as expected and help?

Reply:
Now set to false.